core/src/test/java/org/testcontainers/containers/GenericContainerTest.java

```diff
@@ -54,6 +53,19 @@ public void shouldReportErrorAfterWait() {
         }
     }
 
+    @Test
+    public void shouldLogImageNameWhenGetDockerImageNameFails() {
+        // A docker image that doesn't exist is enough.  The NotFoundException
+        // that the ContainerFetchException wraps may contain the image name.
+        // This test verifies that the ContainerFetchException itself does.
+        String imageName = "doesnotexist";
+        try(GenericContainer container = new GenericContainer<>(imageName)) {
```

Comment:
Can also be `new RemoteDockerImage().resolve()` (and moved to `RemoteDockerImageTest`), no need to use `GenericContainer`

Reply:
Done in https://github.com/testcontainers/testcontainers-java/pull/2450/commits/47329ba7747df814e8d10615eeb474af8b84cc03 I think, though instead of using `resolve()` I'm using `toString()` which seems to be enough.